### ISSUE TITLE
Remove all instances of testing/comparing tr() strings

### ISSUE THREAD

**Issue by jaakristioja** [bibletime/bibletime#82]
All logic similar to `if (something == tr("something")` or `if (list.contains(tr("something")))` is usually very unreliable and should be removed, because it stinks of using unreliable `tr()` strings for passing internal state.

**Comment by gholmlund** [comment]
I did both regular expression searchs and manual inspection of all '==' occurrences. I did not find any problems. Closing.

**Comment by jaakristioja** [comment]
Thanks! :-)

**Comment by jaakristioja** [comment]
Actually, there still were some. I fixed those in BtModuleChooserButton in 1bded2c3:

```diff
-    // Disable non-Bible categories on first button
-    if (m_moduleType == CSwordModuleInfo::Bible && m_id == 0) {
-        QList<QAction*> actions = m_popup->actions();
-        for (int i=0; i<actions.count(); i++) {
-            QAction* action = actions.at(i);
-            QString text = action->text();
-            if (text != QObject::tr("Bibles")) {
-                action->setDisabled(true);
-            }
-        }
-    }
```

But there are also two occurrences in a similar section in [src/frontend/displaywindow/bttextwindowheaderwidget.cpp](https://github.com/bibletime/bibletime/blob/master/src/frontend/displaywindow/bttextwindowheaderwidget.cpp#L113):

```C++
    // Disable non-Bible categories on left replace menu
    if (m_moduleType == CSwordModuleInfo::Bible && m_id == 0) {
        QList<QAction*> actionsType = m_popup->actions();
        for (int t=0; t<actionsType.count(); t++) {
            QAction* actionType = actionsType.at(t);
            QString typeText = actionType->text();
            if (typeText != QObject::tr("Replace"))
                continue;
            QMenu* menuType = actionType->menu();
            if (menuType == nullptr)
                continue;
            QList<QAction*> actions = menuType->actions();
            for (int i=0; i<actions.count(); i++) {
                QAction* action = actions.at(i);
                QString text = action->text();
                if (text != QObject::tr("Bibles")) {
                    action->setDisabled(true);
                }
            }
        }
    }
```

For 3.0 I hope to get around refactoring the BtTextWindowHeaderWidget class to use BtBookshelfModel as well, similarly to what I did in 1bded2c3.

**Comment by jaakristioja** [comment]
Fixed by commit 6e57e29caacf2940fea7844be7c8f23a9da0e9e3.
